paper/paper_quarto.qmd

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
is this autogenerated from the md or do we need to copy paste this every time?

Reply:
the yaml preamble is different than the `paper.md` but the rest is exactly the same. So we can just copy-paste in the content of the matter and leave the preamble alone

Reply:
Yea, saw it was stated in the readme so that is totally fine